### PR TITLE
Add New API Endpoints

### PR DESCRIPTION
**Pull request by CallumAM**
SqlDBM has added more endpoints since this library was created, and we'd like to use some.

I've added 2 of the new end points here (there's 4 left, but wanted to keep my first PR small). Have tested manually, both new functions working as expected in CLI and when using lib. Happy to add automated tests, but will wait to hear back from you first.

Anything you need me to add let me know. Once this PR is in I'll add the next 4, if you'd rather they all go in as one release let me know and can do that too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ pip install pysqldbm[cli]
 ```bash
 sqldbm --help
 
-export SQLDB_API_KEY="your_api_key"
+export SQLDBM_API_KEY="your_api_key"
```
Comment (CallumAM) [inline README.md:20]: Wee typo, did double check that SQLDB_API_KEY does *not* work.

File: src/pysqldbm_cli/run.py [hunk 2]
```diff
@@ -115,6 +114,35 @@ def get_alter_statement(
         print_json({"statement": statement})
 
 
+@run.command("get-latest-object-ddl")
+@click.option("--project-id", type=int, required=True)
+@click.option("--object-name", type=str, required=True)
+@click.option("--case-sensitive", is_flag=True, default=False)
+@click.option("--raw/--no-raw", default=False)
```
Comment (CallumAM) [inline src/pysqldbm_cli/run.py:121]: Added `--case-sensitive` as a flag, which I think fits the spirit of the arg (it's optional in the API string too), but inconsistent with the `--raw/--no-raw` flag. Happy to change back for consistency?

Comment (drewsonne) [reply src/pysqldbm_cli/run.py:121]: What makes it inconsistent? Do we need a warning or a check for mutually exclusive args?

**Review by drewsonne — APPROVED** [review]
I can ask dumb questions if they help, but it lgtm :shipit: 

i no longer have access to sqldbm to test

**Comment by drewsonne** [comment]
This should merge into develop first and then marge to main and push a tag to release to prod

**Comment by CallumAM** [comment]
> I can ask dumb questions if they help, but it lgtm :shipit:
> 
> i no longer have access to sqldbm to test

Tested against SqlDBM, it's all working, and ask away! I'm comfortable writing python but DE does look quite different to this, plenty of things I could miss.

File: src/pysqldbm_cli/run.py (hunk 2 above)
Comment (CallumAM) [reply src/pysqldbm_cli/run.py:121]: I'm new to CLI tools, or at least ones that are used by other people, so if there's a standard pattern to this I don't know it.
But, my feeling for it being incosistent was it's a true/false input in the same way that raw/not raw is. But whereas that's got a got a true and false input this is just a flag.
For mutually exclusive args, it's just raw/not raw right? I can add a check in my next PR (need to add the DBT yaml calls too). Is it work just picking one way and sticking to it though? Quite happy to change this to `--case-sensentive/--case-insensivite` args.

Comment (drewsonne) [reply src/pysqldbm_cli/run.py:121]: oh I see now. Tbh, I've probably got the raw/no-raw wrong. There's some good stuff in here: https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags

I think the way you've done it is right.

**Comment by drewsonne** [comment]
I set up pypi (which you should now have access to) and test pypi (I'd still need your username on there) to use [Trusted Publishers](https://docs.pypi.org/trusted-publishers/) which means there's no more credentials to pass around, so you should have more control. 

Feel free to merge into dev and create a PR for main when you're ready. This LGTM.

I would like to automate this version release process, so please bear with me as we do this the "proper" way the first time, where I can't fiddle with it manually to coax a release. If you don't mind have a looking at #14 that might be worth doing before this PR. I'm watching this repo a bit closer now, so you shouldn't have delays.

**Pull request merged**